core/server/master/src/main/java/alluxio/master/meta/DefaultMetaMaster.java

```diff
@@ -167,10 +164,10 @@ public Address getFieldValue(MasterInfo o) {
   @Override
   public Map<String, TProcessor> getServices() {
     Map<String, TProcessor> services = new HashMap<>();
-    services.put(Constants.META_MASTER_CLIENT_SERVICE_NAME,
-        new MetaMasterClientService.Processor<>(new MetaMasterClientServiceHandler(this)));
-    services.put(Constants.META_MASTER_MASTER_SERVICE_NAME,
-        new MetaMasterMasterService.Processor<>(new MetaMasterMasterServiceHandler(this)));
+    //services.put(Constants.META_MASTER_CLIENT_SERVICE_NAME,
```

Comment:
TODO

Reply:
Being addressed as part of boot-up and security work.